master/buildbot/db/model.py

```diff
@@ -304,6 +308,7 @@ class Model(base.DBConnectorComponent):
     # Files touched in changes
     change_files = sautils.Table(
         'change_files', metadata,
+        sa.Column('dummy', sa.Integer, primary_key=True, autoincrement=True),
```

Comment:
filename is a VARCHAR(1024), that is too long to be a (primary) key:

> Got fatal OperationalError on DB
> Traceback (most recent call last):
> Failure: sqlalchemy.exc.OperationalError: (_mysql_exceptions.OperationalError) (1071, 'Specified key was too long; max key length is 3072 bytes') [SQL: u'\nCREATE TABLE change_files (\n\tchangeid INTEGER NOT NULL, \n\tfilename VARCHAR(1024) NOT NULL, \n\tPRIMARY KEY (changeid, filename), \n\tFOREIGN KEY(changeid) REFERENCES changes (changeid)\n)CHARACTER SET utf8\n\n']

https://dev.mysql.com/doc/refman/5.7/en/innodb-restrictions.html